src/App/Datums.cpp

```diff
@@ -129,16 +129,18 @@ Point::Point()
 // ----------------------------------------------------------------------------
 
 LocalCoordinateSystem::LocalCoordinateSystem()
-    : extension(this)
 {
     ADD_PROPERTY_TYPE(OriginFeatures,
                       (nullptr),
                       0,
                       App::Prop_Hidden,
                       "Axis and baseplanes controlled by the LCS");
 
+    Group.setStatus(Property::Transient, true);
```

Comment:
Why is this property set to Transient? It seems like quite odd choice. I see that later you explain in comment that LCS does not store stuff inside group property but OriginFeatures - this might be good to put here in comment as well.

Reply:
I do not know. This was actually there before, but it was in the extension that was bundled in the LCS. I am not sure if I was correct porting it, why it was here and if it should be removed.
If you have a suggestion what to do about it I'm all ears

Reply:
If it was in the extension I think it is fair to simply move it - it would be great to comment on it but if you are not sure about intent either I think we should simply leave it as is. It is better to leave something not documented than to spread misinformation in documentation.